test/features/email/presentation/controller/single_email_controller_test.dart

```diff
@@ -252,4 +261,39 @@ void main() {
       },
     );
   });
+
+  group('calendar event reply test:', () {
+    final blobId = Id('abc123');
+    final calendarEvent = CalendarEvent();
+
+    setUp(() {
+      when(mailboxDashboardController.selectedEmail).thenReturn(Rxn(null));
+      when(mailboxDashboardController.emailUIAction).thenReturn(Rxn(null));
+      when(mailboxDashboardController.viewState).thenReturn(Rx(Right(UIState.idle)));
```

Comment:
IMO, do not use `when` in `setUp` method

Reply:
the test won't pass if I put it inside `test` function

Reply:
should find the root cause. IMO, should use in `test` only

Reply:
Will try and fixup with update